web/src/main/java/org/mskcc/cbio/oncokb/controller/DriveAnnotationParser.java

```diff
@@ -688,12 +672,14 @@ private void parseTherapeuticImplications(Gene gene, Set<Alteration> alterations
             addDateToLastEditSetFromObject(lastEditDates, drugObj, "name");
 //            addDateToLastReviewSetFromLong(lastReviewDates, drugObj, "name");
 
+            Pair<EvidenceType, String> evidenceTypeAndKnownEffect =  getEvidenceTypeAndKnownEffectFromDrugObj(drugObj);
+
             Evidence evidence = new Evidence();
-            evidence.setEvidenceType(evidenceType);
+            evidence.setEvidenceType(evidenceTypeAndKnownEffect.getKey());
```

Comment:
do you wanna check null?  evidenceTypeAndKnownEffect can be null based on method `getEvidenceTypeAndKnownEffectFromDrugObj`